Cargo.toml

```diff
@@ -13,3 +13,6 @@ members = [
     "arrow-udf-python",
     "arrow-udf-wasm",
 ]
+
+[patch.crates-io]
+bit-set = { git = "https://github.com/contain-rs/bit-set.git" }
```

Comment:
Let's temporarily patch it.

Reply:
Should be fixed in https://github.com/risingwavelabs/arrow-udf/pull/39